motor/discovery/gcp/mql_asset_objects.go

```diff
@@ -38,9 +45,92 @@ func getTitleFamily(o gcpObject) (gcpObjectPlatformInfo, error) {
 	return gcpObjectPlatformInfo{}, errors.Newf("missing runtime info for gcp object service %s type %s", o.service, o.objectType)
 }
 
-func computeImages(m *MqlDiscovery, project string, tc *providers.Config) []*asset.Asset {
+func computeInstances(m *MqlDiscovery, project string, tc *providers.Config, sfn common.QuerySecretFn) ([]*asset.Asset, error) {
 	assets := []*asset.Asset{}
-	images := m.GetList("return gcp.project.compute.images { id name labels }")
+
+	instances, err := m.GetList("return gcp.project.compute.instances { id name labels zone { name } status networkInterfaces disks { guestOsFeatures } }")
+	if err != nil {
+		return nil, err
+	}
+	for i := range instances {
+		b := instances[i].(map[string]interface{})
+		id := b["id"].(string)
+		name := b["name"].(string)
+
+		disks := b["disks"].([]interface{})
+		if disksContainWindows(disks) {
+			log.Debug().Msgf("skipping windows instance %s", name)
+			continue
+		}
+
+		tags := b["labels"].(map[string]interface{})
+		zone := b["zone"].(map[string]interface{})
+		zoneName := zone["name"].(string)
+		status := b["status"].(string)
+		stringLabels := make(map[string]string)
+		for k, v := range tags {
+			stringLabels[k] = v.(string)
+		}
+		stringLabels["mondoo.com/instance"] = id
```

Comment:
nit: Other labels have a `const` in `motor/discovery/gcp/consts.go`. Would it help to also have one for this label?

Reply:
Yeah, this one seemed to be a global label (not GCP-specific) but I will add it to that list too